src/python/bcc/libbcc.py

```diff
@@ -231,6 +231,10 @@ class bcc_symbol_option(ct.Structure):
             ('use_symbol_type', ct.c_uint),
         ]
 
+lib.bcc_procutils_which_so_in_process.restype = ct.POINTER(ct.c_char)
+lib.bcc_procutils_which_so_in_process.argtypes = [ct.c_char_p, ct.c_int]
+lib.bcc_procutils_which_so_in_ldconfig_cache.restype = ct.POINTER(ct.c_char)
+lib.bcc_procutils_which_so_in_ldconfig_cache.argtypes = [ct.c_char_p]
```

Comment:
Are the above lib.bcc_procutils_which_so_in_ldconfig_cache necessary or not? find_library(libname, 0) should cover this.

Reply:
Fair enough! No need to expose this function.